### PR TITLE
fix: DEV-1943: After awhile, shapes disappear

### PR DESCRIPTION
**Pull request by yyassi-heartex**

### CODE REVIEW

File: src/components/Toolbar/Tool.js [hunk 1]
```diff
@@ -70,14 +70,15 @@ export const Tool = ({
     currentShortcut = shortcut;
     if (shortcut && !hotkeys.hasKey(shortcut)) {
       hotkeys.addKey(shortcut, () => {
-        onClick?.();
+        if (!(tool?.annotation?.isDrawing ?? false))
```
Comment (bmartel) [inline src/components/Toolbar/Tool.js:73]:
```suggestion
        if (!(tool?.annotation?.isDrawing))
```

Comment (bmartel) [reply src/components/Toolbar/Tool.js:73]: its already boolean coerced, no need for the null coalesce.

File: src/components/Toolbar/Tool.js [hunk 2]
```diff
@@ -106,17 +107,20 @@ export const Tool = ({
   }, [smart, extra]);
 
   const showControls = dynamic === false && controls?.length && (active || (controlsOnHover && hovered));
+  const isAnnotationDrawing = tool?.annotation?.isDrawing ?? false;
```
Comment (bmartel) [inline src/components/Toolbar/Tool.js:110]:
```suggestion
  const isAnnotationDrawing = tool?.annotation?.isDrawing;
```
same here

**Review by bmartel — CHANGES_REQUESTED** [review]
Looks good, couple minor observations.

**Review by bmartel — APPROVED** [review]
Looks good, checked it and it works well 💪

**Comment by hlomzik** [comment]
> /git merge develop
there is no `develop` in LSF :)

**Comment by yyassi-heartex** [comment]
> > /git merge develop
> 
> there is no `develop` in LSF :)

lol yep as soon as i typed it im like "oh right" so i merged manually lol

File: src/tools/Base.js [hunk 3]
```diff
@@ -24,13 +24,17 @@ const ToolView = observer(({ item }) => {
...
     smart: false,
+    isDrawingTool: false,
   })
   .volatile(() => ({
     dynamic: false,
     index: 1,
   }))
   .views(self => {
     return {
+      // get isDrawingTool() {
```
Comment (bmartel) [inline src/tools/Base.js:35]: I think this was meant to be removed

**Review by bmartel — APPROVED** [review]
Just the one commented out bit needs to get removed, approving this on that removal.

File: src/tools/Base.js (hunk 3 above)
Comment (yyassi-heartex) [reply src/tools/Base.js:35]: you are right - i'll get rid of it

**Pull request merged**